sdcm/utils/k8s/chaos_mesh.py

```diff
@@ -61,15 +62,17 @@ class ChaosMesh:  # pylint: disable=too-few-public-methods
 
     def __init__(self, k8s_cluster: "sdcm.cluster_k8s.KubernetesCluster"):
         self._k8s_cluster = k8s_cluster
+        self.log = SDCMAdapter(LOGGER, extra={'prefix': k8s_cluster.region_name})
```

Comment:
was there a problem with using `self._k8s_cluster.log` method?

Reply:
> was there a problem with using `self._k8s_cluster.log` method?

It just will refer to different module, because it is initialized using `__name__`.